plugins/woocommerce-admin/client/analytics/components/leaderboard/index.js

```diff
@@ -38,18 +79,7 @@ export class Leaderboard extends Component {
 
 	getFormattedRows() {
 		return this.props.rows.map( ( row ) => {
-			return row.map( ( column ) => {
-				return {
-					display: (
-						<div
-							dangerouslySetInnerHTML={ sanitizeHTML(
-								column.display
-							) }
-						/>
-					),
-					value: column.value,
-				};
-			} );
+			return row.map( this.getFormattedColumn.bind( this ) );
```

Comment:
This creates a new function for every `row`, I think we can make it so we'll create a new function for each `getFormattedRows` call:
```js
		const getFormattedColumn = this.getFormattedColumn.bind( this );

		return this.props.rows.map( ( row ) => {
			return row.map( getFormattedColumn );
```

Reply:
Thanks for the suggestion. Adjusted in https://github.com/woocommerce/woocommerce/pull/49097/commits/0cf5d717b9442251f19a24e8f630cd0ff69e2b3c.